### PR TITLE
#issue-23245: Changing access token renewal job trigger name and grou…

### LINKED ISSUE #23245: Create Analytics tokens refresh job
In order to keep DotCMS up with the Access Token generation to access the analytics stack we need a scheduled job.
This job will be in charge of:
- Running every now and then (frequency TBD) to detect when an Analytics Access Token is near to expire
- When the token is within in a configurable window of time, the job will try to refresh it in s separate thread
- During this window of time any other requests can consume the not-yet-expired token in order to avoid any blocking situation
- Retry a TBD number of times when the access token request fails
- Detect when Analytics stack is unrecoverable and set DotCMS in a state that it can notify that access tokens can no longer be refreshed

Flow for Access Token Request

* Is Auth in an unrecoverable error state, e.g. invalid secure secret, missing details
     * Return exception, need to direct to configuration page to update security.
* Is token in 1 min (configurable) window of expiry and update request thread (should not need a dedicated thread for this 
   just a background job in executor but should not run in parallel) not already running
	* yes
	  *  spawn thread to update token
			* On success
             	               * Update cached token
			* On failure
			       * If temporary error, e.g. network connection error, recoverable retry max 2 (delay ?) ?
                                * If a  permanent error, e.g. secret is wrong report  error notification exception, set status, do not keep retrying block further requests.
          * return old token immediately.
     * No Expired
		* spawn thread to update token if thread not already running
		* block for new token ( timeout auth exception, timeout must take into account any retry and delay, and preferably not fully block entire system, maybe add spinner until this timeout )
     * No
		* return existing token from cache no block.

### PR DESCRIPTION
**Pull request by victoralfaro-dotcms**
Adding some flags for testing purposes and using another trigger and group name for job since `trigger30` was already taken.

### CODE REVIEW

**Review by freddyucv — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/analytics/AnalyticsAPIImpl.java [hunk 1]
```diff
@@ -259,7 +266,7 @@ private void logTokenResponse(final CircuitBreakerUrl.Response<AccessToken> resp
      */
     private Map<String, String> accessTokenHeaders(final AnalyticsApp analyticsApp) {
         return ImmutableMap.<String, String>builder()
-            .put(HttpHeaders.AUTHORIZATION, String.format("Basic %s", analyticsApp.clientIdAndSecret()))
+            //.put(HttpHeaders.AUTHORIZATION, String.format("Basic %s", analyticsApp.clientIdAndSecret()))
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/analytics/AnalyticsAPIImpl.java:269]: remove comment?

File: dotCMS/src/curl-test/pp-chepi.json [hunk 2]
```diff
@@ -0,0 +1,4443 @@
+{
+	"info": {
```
Comment (dsilvam) [inline dotCMS/src/curl-test/pp-chepi.json:2]: why is this called pp-chepi?

File: dotCMS/src/main/java/com/dotcms/analytics/AnalyticsAPIImpl.java (hunk 1 above)
Comment (victoralfaro-dotcms) [reply dotCMS/src/main/java/com/dotcms/analytics/AnalyticsAPIImpl.java]: Removed

File: dotCMS/src/curl-test/pp-chepi.json (hunk 2 above)
Comment (victoralfaro-dotcms) [reply dotCMS/src/curl-test/pp-chepi.json]: File removed.

**Review by dsilvam — APPROVED** [review]

**Pull request merged**